environment_and_metadata.rst

```diff
@@ -749,12 +749,12 @@ the SIF file metadata descriptor.
     ├── libs
     ├── runscript
     ├── runscript.help
-    ├── Singularity
+    ├── {Singularity}
```

Comment:
No sure if this changes... just built a container and checked.

```sh
➜  .singularity.d sn --version
singularity-ce version 3.8.0-rc.2
➜  .singularity.d pwd
/tmp/debian/.singularity.d
➜  .singularity.d ll
total 28K
drwxr-xr-x 2 wobito wobito 4.0K May 20 12:37 actions
drwxr-xr-x 2 wobito wobito 4.0K May 20 12:37 env
-rw-r--r-- 1 wobito wobito  352 May 20 12:37 labels.json
drwxr-xr-x 2 wobito wobito 4.0K May 20 12:37 libs
-rwxr-xr-x 1 wobito wobito 1.1K May 20 12:37 runscript
-rw-r--r-- 1 wobito wobito   39 May 20 12:37 Singularity
-rwxr-xr-x 1 wobito wobito   10 May 20 12:37 startscript
➜  .singularity.d cat Singularity
bootstrap: docker
from: debian:latest
```

Reply:
👍 good spot, thanks. I'll take care of this!

Reply:
The file tree is still appropriate for a container built out with all the possible def file sections.